sklearn/ensemble/_forest.py

```diff
@@ -193,7 +193,8 @@ def __init__(self,
                  verbose=0,
                  warm_start=False,
                  class_weight=None,
-                 max_samples=None):
+                 max_samples=None,
+                 importance_type='impurity'):
```

Comment:
We need to have under `self.importance_type = importance_type` (or whatever named. It means that we will share the parameter between RandomForest, RandomTreesEmbedding, and ExtraTrees.

Reply:
Added